README.md

```diff
@@ -71,6 +71,7 @@ Creates a new bossbat instance. All arguments are optional.
 - `options.prefix`: A string that all redis keys will be prefixed with. Defaults to `bossbat`.
 - `options.ttl`: The number of milliseconds before a job times out. Setting it will change the maximum duration that jobs can hold a lock. By default, job locks will timeout if a job does not complete in 2000ms.
 - `options.tz`: An optional timezone used with `jobDefinition.cron` expressions.
+- `options.modifyConfig`: Should the redis `CONFIG` command be used. Defaults to `true`
```

Comment:
I'd prefer an option that defaults to false. Maybe `disableRedisConfig`, which defaults to false. Maybe also put a brief explanation here of why the config command is used at all.